src/CoverCrypt.tsx

```diff
@@ -383,13 +364,18 @@ const CoverCrypt: React.FC<{ kmsToken: string }> = ({ kmsToken }) => {
 
       <Stack spacing={3}>
         <Text fontSize="md">
-          Consider the following 2 policy axes, Department and Country which data are partitioned by the following attributes:
+          Consider 2 policy axes: <Text as="b">Department</Text> and <Text as="b">Country</Text>. Each axes are partitioned by attributes:{" "}
+          <Text as="b">Marketing and HR</Text> for the Departement axe and <Text as="b">France, Spain and Germany</Text> for the Country
```

Comment:
typo - <Text as="b">Marketing and HR</Text> for the Departement **axis** and <Text as="b">France, Spain and Germany</Text> for the Country **axis**